minecraft_servers/risingpixels/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "server_name": "risingpixels",
+  "nice_name": "RisingPixels",
+  "direct_ip": "risingpixels.net",
+  "server_wildcards": [
+    "player.risingpixels.net"
```

Comment:
That's not a wildcard
https://en.wikipedia.org/wiki/Wildcard_DNS_record

Reply:
Removed wildcards now :)